pkgs/development/python-modules/pysparse/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, setuptools
+, liblapack
+, blas
+, isPy27
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "pysparse";
+  version = "1.3-dev";
+  disabled = !isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1d5dw3cyf4b58rw72x5kg82j7x19dkiq6axffsgzyf7xj0vny1lr";
+  };
+
+  hardeningDisable = [ "all" ];
+
+  propagatedBuildInputs = [
+    numpy
+    blas
```

Comment:
Does it need to link to the same blas as is used by numpy? There is the `numpy.blas` attribute.

Reply:
Good catch, yes it does.

Reply:
Now using `numpy.blas`